allennlp/fairness/bias_mitigator_wrappers.py

```diff
@@ -0,0 +1,266 @@
+import torch
+from typing import Union
+from os import PathLike
+
+from allennlp.fairness import (
+    HardBiasMitigator,
+    LinearBiasMitigator,
+    INLPBiasMitigator,
+    OSCaRBiasMitigator,
+)
+from allennlp.fairness.bias_direction_wrappers import BiasDirectionWrapper
+from allennlp.fairness.bias_utils import load_word_pairs
+
+from allennlp.common import Registrable
+from allennlp.data.tokenizers.tokenizer import Tokenizer
+from allennlp.data import Vocabulary
+
+
+class BiasMitigatorWrapper(Registrable):
+    """
+    Parent class for bias mitigator wrappers.
+    """
+
+    def train(self, mode: bool = True):
+        """
+
+        # Parameters
+
+        mode : `bool`, optional (default=`True`)
+            Sets `requires_grad` to value of `mode` for bias mitigator
+            and associated bias direction.
+        """
+        raise NotImplementedError
+
+
+# TODO: remove equalize words from evaluation words
```

Comment:
Is this for later, or for this PR?

Reply:
This is for later, the current assumption is that the user won't have any overlap between the embeddings for which to mitigate bias and the equalize set.